docs/dunst.5.pod

```diff
@@ -389,8 +388,43 @@ If B<icon_position> is set to off, this setting is ignored.
 Can be set to a colon-separated list of paths to search for icons to use with
 notifications.
 
-Dunst doesn't currently do any type of icon lookup outside of these
-directories.
+Dunst doens't search outside of these direcories. For a recursive icon lookup
+system, see B<enable_recursive_icon_lookup>. This new system will eventually
+replace this and will need new settings.
+
+=item B<icon_theme> (default: "Adwaita", example: "Adwaita, breeze") I<Experimental>
+
+Comma-separated of names of the the themes to use for looking up icons. This has
+to be the name of the directory in which the theme is located, not the
+human-friendly name of the theme. So for example, the theme B<Breeze Dark> is
+located in F</usr/share/icons/breeze-dark>. In this case you have to set the
+theme to B<breeze-dark>.
+
+The first theme in the list is the most important. Only if the icon cannot be
+found in that theme, the next theme will be tried.
+
+Dunst will look for the themes in B<XDG_DATA_HOME/icons> and
+B<$XDG_DATA_DIRS/icons> as specified in the icon theme specification:
+https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html.
+
+If the theme inherits from other themes, they will be used as a backup.
+
+This setting is experimental and not enabled by default. See
+B<enable_recursive_icon_lookup> for how to enable it.
+
+=item B<enable_recursive_icon_lookup> (default: false) I<Experimental>
+
+This setting enables the new icon lookup method. This new system will eventually
+be the old icon lookup.
+
+Currently icons are looked up in the B<icon_path> and scaled according to
+B<min_icon_size> and B<max_icon_size>. Since the B<icon_path> wasn't recursive,
+one had to add a ton of paths to this list.
+This has been drastically simplified by the new lookup method. Now you only have
+to set B<icon_theme> to the name of the theme and B<icon_size> to the icon size
+you want. To enable this new behaviour, set B<enable_recursive_icon_lookup> to
+true in the I<[experimental]> section. See the respective settings for more
```

Comment:
This seems wrong, it expects it in the `global` section

Reply:
Experimental settings are no longer in the experimental section. This is because some of them are rules, so that wouldnt make sense